gentoopm/bash/bashserver.py

```diff
@@ -37,23 +37,23 @@ def __del__(self):
 		self._bashproc.communicate()
 
 	def load_file(self, envf):
-		f = tempfile.NamedTemporaryFile('w+b')
```

Comment:
Could you split irrelevant `with` conversion to a separate commit? It makes it hard to figure out what you changed.